va/x11/va_x11.c

```diff
@@ -139,7 +153,11 @@ static VAStatus va_DisplayContextGetDriverName (
 	*driver_name = NULL;
     else
         return VA_STATUS_ERROR_UNKNOWN;
-    
+
+#ifdef HAVE_VA_DRI3
+    vaStatus = va_DRI3GetDriverName(pDisplayContext, driver_name);
+    if (vaStatus != VA_STATUS_SUCCESS)
+#endif
     vaStatus = va_DRI2GetDriverName(pDisplayContext, driver_name);
     if (vaStatus != VA_STATUS_SUCCESS)
```

Comment:
It will have compatibility issue if the driver supports DRI2 only but both DRI3 and DRI2 can work.

Reply:
Shouldn't VA_STATUS_SUCCESS be false if the driver doesn't support DRI3 or does va_DRI3GetDriverName() always return true if the X server supports DRI3 but the driver doesn't?